src/ai/backend/manager/config.py

```diff
@@ -256,6 +256,7 @@
                     t.Key("importer-image", default="lablup/importer:manylinux2010"): t.String,
                     t.Key("max-wsmsg-size", default=16 * (2**20)): t.ToInt,  # default: 16 MiB
                     t.Key("aiomonitor-port", default=50100): t.Int[1:65535],
+                    t.Key("use-host-network", default=False): t.Bool,
```

Comment:
Since the configuration knob is moved to the scaling groups, let's remove this field.